### ISSUE TITLE
Invalid argument supplied for foreach() in f3/db/mongo/mapper.php:87

### ISSUE THREAD

**Issue by francodgstn** [bcosca/fatfree#781]
When I try to load a "record" (mongo document) using something like this:

```php
$db = new DB\Mongo('mongodb://localhost:27017','mydb'); 
$person = new DB\Mongo\Mapper($db,'people');
$person->load(array("email" => "theemail@email.me" ));
```
It get me an __Error 500 Invalid argument supplied for foreach()__ in f3/db/mongo/mapper.php:87.

I already reported the bug on Google Groups https://groups.google.com/forum/#!topic/f3-framework/nqeQKQb3V7E

**Comment by francodgstn** [comment]
The bug only affects windows most likely due to the mongo driver, I tried the same code on a linux enviroment and it works well.

**Comment by ptdev** [comment]
Also getting this error on linux with php 5.5.21 and 5.6.5.
The fix described at [#776](https://github.com/bcosca/fatfree/issues/776) seems to solve the issue for me.
